packages/hardhat/contracts/Ethernauts.sol

```diff
@@ -23,8 +23,8 @@ contract Ethernauts is ERC721Enumerable, Ownable {
     error TokensGiftError(uint256 gifted, uint256 maxToGift);
     error PermanentUrlError(bool permanentURI);
     error CurrentStateError(SaleState currentState, SaleState availableState);
-    error SaleStateError(SaleState newState, SaleState current, SaleState completed);
-    error RecoverTokenError(address tokenAddress, address toAddress);
+    error SaleStateError(SaleState newState, SaleState current);
+    error RecoverTokenError(string message);
```

Comment:
here, let's let the address tokenAddress, address toAddress back and modify the test to meet this, maybe put a generic error on the test